app/src/main/java/org/oppia/app/player/content_interaction/ContentInteractionFragment.kt

```diff
@@ -0,0 +1,26 @@
+package org.oppia.app.player.content_interaction
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.app.fragment.InjectableFragment
+import org.oppia.app.player.content_interaction.ContentInteractionFragmentPresenter
+import javax.inject.Inject
+
+/** Fragment displays single/multiplechoice input interaction. */
```

Comment:
Do not you `single` or `singlechoice` or `singlechoiceinput` anywhere, in our discussion that was just for communication.
If you are using `SingleChoiceInput` -> that by default developer will understand that in this radio buttons should be visible and leaner can select only one correct answer. This then drives the assumption that `MultipleChoiceInput` will show checkboxes and learner can select multiple answers.

This is incorrect in terms of definition as well as oppia-web codebase.

Use `ItemSelectionInput` or `ItemSelectionViewInteraction` whichever fits the name, this means that learner can select multiple answers at a time in backend code.
Use `MultipleChoiceInput` or `MultipleChoiceViewInteraction`, this means that learner can select one answer among multiple options, as per backend code.

Reply:
this file is been removed